config/initializers/inflections.rb

```diff
@@ -1,8 +1,38 @@
 # Be sure to restart your server when you modify this file.
 
+Rails.autoloaders.each do |autoloader|
+  autoloader.inflector.inflect(
+    'proxy_api' => 'ProxyAPI'
+  )
+end
+
 # Add new inflection rules using the following format
 # (all these examples are active by default):
 ActiveSupport::Inflector.inflections do |inflect|
+  inflect.acronym 'AAAA' # DNS IPv6 record
+  inflect.acronym 'AIX' # IBM AIX
+  # Requires our entire Api namespace to be rewritten
+  # inflect.acronym 'API' # Application Programmer Interface
+  inflect.acronym 'AWS' # Amazon Web Services
+  inflect.acronym 'BMC' # Baseboard Management Controller
+  inflect.acronym 'CA' # Certificate Authority
+  inflect.acronym 'DHCP' # Dynamic Host Configuration Protocol
+  inflect.acronym 'DNS' # Domain Name System
+  inflect.acronym 'EC2' # Amazon Elastic Compute Cloud
+  inflect.acronym 'GCE' # Google Compute Engine
+  inflect.acronym 'IPAM' # IP Address Management
+  inflect.acronym 'NXOS' # Cisco NX-OS
+  inflect.acronym 'PTR' # DNS reverse DNS record
+  # Causes an overlap between ::SSHKey and the SshKey model
+  # inflect.acronym 'SSH' # Secure SHell
```

Comment:
While not ideal, could we remove the overlap by for example nesting our `::SshKey` in a module? If github search can be trusted then the `::SshKey` class doesn't seem to be used anywhere outside of this repo so the rename should be realtively trouble-free.

Reply:
Do you also propose to change the DB model name?